modules/brushingandlinking/src/datastructures/brushingaction.cpp

```diff
@@ -52,4 +53,50 @@ std::string_view BrushingTarget::findOrAdd(std::string_view target) {
     }
 }
 
+std::string_view enumToStr(BrushingAction action) {
```

Comment:
Github actions windows build: is the warning something to care about?
> GitHub Actions
/ build (windows-latest)
C4273: 'inviwo::enumToStr': inconsistent dll linkage [D:\a\inviwo\inviwo\build\modules\brushingandlinking\inviwo-module-brushingandlinking.vcxproj]